tests/SeoBundle/RedirectHandlerTestCase.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace SeoBundle;
+
+use Pimcore;
+use Pimcore\Bundle\SeoBundle\Redirect\RedirectHandler;
+use Pimcore\Tests\Support\Test\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+
+class RedirectHandlerTestCase extends TestCase
```

Comment:
Maybe by 

```suggestion
namespace Pimcore\Tests\SeoBundle;

use Pimcore;
use Pimcore\Bundle\SeoBundle\Redirect\RedirectHandler;
use Pimcore\Tests\Support\Test\TestCase;
use Symfony\Component\HttpFoundation\Request;

class RedirectHandlerTest extends TestCase
```
and also renaming the file

Reply:
or maybe the entire SeoBundle folder just need to be under `test/Model` folder, don't know if it can detect the tests outside of it